backend/app/notify/notify.go

```diff
@@ -94,22 +97,27 @@ func (s *Service) Submit(req Request) {
 	}
 }
 
-// getNotificationEmails returns list of emails for notifications for provided comment.
-// Emails is not added to the returned list in case original message is from the same user as the notification receiver.
-func (s *Service) getNotificationEmails(req Request, notifyComment store.Comment) (result []string) {
+// getNotificationDetails returns list of details (like email or telegram username) for users
+// interested in notifications for provided comment.
+// Details are not added to the returned list in case the original message is from the same user as the notification receiver.
+func (s *Service) getNotificationDetails(
+	req Request,
+	notifyComment store.Comment,
+	getUserDetail func(string, string) (string, error),
+) (result []string) {
```

Comment:
this func is a little bit confusing. Ptobably definig a type for this getUserDetail will make it better looking. Also naming - it doesn't return much of the details but rather a list of destinations/targets

Reply:
Renamed, moved `getUserDetail` to a separate type with a comment.